test/blocks/global-footer/global-footer.test.js

```diff
@@ -0,0 +1,263 @@
+import { expect } from '@esm-bundle/chai';
+import sinon, { stub } from 'sinon';
+import {
+  allElementsVisible,
+  alwaysVisibleSelectorsDesktop,
+  alwaysVisibleSelectorsMobile,
+  containerSelector,
+  createFullGlobalFooter,
+  insertDummyElementOnTop,
+  selectors,
+  waitForFooterToDecorate,
+} from './test-utilities.js';
+import baseFooter from './mocks/base-footer.js';
+import fetchedFooter from './mocks/fetched-footer.js';
+import icons from './mocks/icons.js';
+import { isElementVisible, mockRes } from '../global-navigation/test-utilities.js';
+import placeholders from './mocks/placeholders.js';
+
+const originalFetch = window.fetch;
+
+describe('global footer', () => {
+  let clock = null;
+  beforeEach(async () => {
+    document.body.innerHTML = baseFooter;
+    clock = sinon.useFakeTimers({
+      toFake: ['setTimeout'],
+      shouldAdvanceTime: true,
+    });
+
+    window.fetch = stub().callsFake((url) => {
+      if (url.includes('/footer')) {
+        return mockRes({
+          payload: fetchedFooter(
+            { regionPickerHash: '/fragments/regions#langnav' },
+          ),
+        });
+      }
+
+      if (url.includes('/placeholders')) return mockRes({ payload: placeholders });
+
+      if (url.includes('icons.svg')) return mockRes({ payload: icons });
+
+      return null;
+    });
+  });
+
+  afterEach(() => {
+    clock.restore();
+    window.fetch = originalFetch;
+  });
+
+  describe('wide screen', async () => {
+    it('should render the footer on wide screen', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true, viewport: 'wide' });
+
+      expect(footer).to.exist;
+
+      Object.keys(selectors).forEach((key) => expect(
+        document.querySelector(selectors[key]) instanceof HTMLElement,
+      ).to.equal(true));
+
+      const elementsAreVisible = allElementsVisible(
+        alwaysVisibleSelectorsDesktop,
+        document.querySelector(containerSelector),
+      );
+      expect(elementsAreVisible).to.equal(true);
```

Comment:
I personally prefer skipping on variables if they don't provide context or are used multiple times.
```suggestion
      expect(allElementsVisible(
        alwaysVisibleSelectorsDesktop,
        document.querySelector(containerSelector),
      )).to.equal(true);
```

Reply:
True, if they are not used multiple times they just add more lines of code for no reason